azurelinuxagent/ga/monitor.py

```diff
@@ -177,11 +180,35 @@ def collect_and_send_events(self):
         except ProtocolError as e:
             logger.error("{0}", e)
 
+    def send_daily_heartbeat(self):
+        """
+        Send the daily heartbeat containing the installed and current agent versions.
+        Send once during the first 30 minutes of the day.
+        """
+        if datetime.datetime.time() <= datetime.time(0, minute=30):
+            try:
+                msg = u"{0}:{1}".format(
+                    INSTALLED_VERSION,
+                    CURRENT_VERSION)
```

Comment:
I consider it, but is a variable amount of data. The underlying goal is to improve basic telemetry so we know what versions are running.